sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -20,11 +26,19 @@ public abstract partial class BinaryContent : System.IDisposable
     }
     public partial class ClientResult
     {
-        protected ClientResult(System.ClientModel.Primitives.PipelineResponse response) { }
+        protected ClientResult(System.ClientModel.Primitives.PipelineResponse? response) { }
```

Comment:
would it be better to add a default constructor? I worry that allowing to pass null says that null is a valid value.

Reply:
Yep, good idea.  I had actually decided to do that for the collection types, but hadn't done a consistency pass yet.  Thanks!

Reply:
Addressed.